src/html_files/utils.ts

```diff
@@ -199,3 +199,7 @@ function get_data_key(data_type, key) {
     }
     return key_value_map;
 }
+
+function percent_difference(v1, v2) {
```

Comment:
v1 and v2 should be typed as `number` correct?  This function should also return a `number`.